src/app/zones/views/ZonesList/ZonesListHeader/ZonesListHeader.tsx

```diff
@@ -20,25 +21,28 @@ const ZonesListHeader = ({
 
   useFetchActions([actions.fetch]);
 
-  const buttons = [
-    <Button
-      data-testid="add-zone"
-      key="add-zone"
-      onClick={() => {
-        setSidePanelContent({ view: ZoneActionSidePanelViews.CREATE_ZONE });
-      }}
-    >
-      Add AZ
-    </Button>,
-  ];
-
   return (
-    <SectionHeader
-      buttons={buttons}
-      subtitle={`${zonesCount} AZs available`}
-      subtitleLoading={!zonesLoaded}
-      title={<ZonesListTitle />}
-    ></SectionHeader>
+    <MainToolbar>
+      <MainToolbar.Title>
+        <ZonesListTitle />
+      </MainToolbar.Title>
+      {zonesLoaded ? (
+        <ModelListSubtitle available={zonesCount} modelName="AZ" />
```

Comment:
Just wanted to note that this results in `AZS available` instead of  `AZs available` when there are multiple zones. our use of pluralize within the `ModelListSubtitle` component may need some adjustments.

Reply:
https://github.com/plurals/pluralize/issues/127